node/cli/src/main.rs

```diff
@@ -17,5 +17,6 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 fn main() -> gear_cli::Result<()> {
+    gear_runtime_interface::sandbox_init();
```

Comment:
sorry can't agree with that. This function should be called only once for the whole process so the place is the most appropriate from this point of view.

Reply:
> This function should be called only once for the whole process so the place is the most appropriate from this point of view.

You can make INIT ONCE if you suppose that our service can run this twice. But making in cli is a not correct way because cli is not the last point. We can make another binaries that works like node and use service

Reply:
I don't get it. Why initialization function wasn't required earlier?

Reply:
Frankly speaking  all this stuff required to init `wasmer` first. The new implementation uses global variable that initializes lazily and hence after our lazy-pages mechanism what leads to failures.

The old implementation relied on the data contained in Caller so `wasmer` got initialized before lazy-pages

Reply:
reverted since lazy-pages fail to initialize properly  - https://github.com/gear-tech/gear/actions/runs/5473227535/jobs/9966425204?pr=2544